src/expr.rs

```diff
@@ -2310,6 +2302,22 @@ where
     }
 }
 
+fn args_have_many_closure<T>(args: &[&T]) -> bool
```

Comment:
Could you add a comment describing what this function does please?

Reply:
Thanks! Added a comment.